### DISCUSSION TITLE
Doc comment on Custom modules (/modules/other-modules/custom-modules)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1829] [Docs/blog comments]
# Doc comment on Custom modules (/modules/other-modules/custom-modules)

Introduction

https://weaviate.io/developers/weaviate/modules/other-modules/custom-modules

**Comment by RolfDeVries** [comment]
Here is a question about: "A. Replace parts of an existing module"
I'm working on a proof of technology for a proprietary LLM. First of all, great job on the documentation, it explains the choices to be made very clearly.

The API seems fair, no problems implementing it, but is a bulk approach also possible? I'm concerned about the overhead of all the https calls. If not, would my concerns be correct or solvable in another way?
